helm/admission-controller/templates/pdb.yaml

```diff
@@ -14,6 +14,5 @@ spec:
   maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
 {{- end }}
   selector:
-    matchLabels:
```

Comment:
Did you remove the matchLabels or is it github thingy?

Reply:
No I haven't touched it.

![image](https://user-images.githubusercontent.com/1936982/90495476-f739af80-e144-11ea-90e3-42097f784c65.png)